scripts/gaussian_codegen/shared/move_emit.py

```
@@ -0,0 +1,131 @@
+    output that a separate `prettier` pass would later reflow (the historical
+    source of the coefficients-file drift).
+
+    Raises if prettier is unavailable, so generation fails loudly rather than
+    emitting unformatted output that would later trip the formatter gate."""
+    prettier = repo_root / "node_modules" / ".bin" / "prettier"
+    if not prettier.exists():
+        raise RuntimeError(
+            f"prettier not found at {prettier}; run `npm ci` at the repo root "
+            "(installs @mysten/prettier-plugin-move, the Move formatter)"
+        )
+    result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)